Pokedex_ios/Helpers/GraphQLHelpers/queries.graphql

```diff
@@ -0,0 +1,21 @@
+query homePage{
```

Comment:
this should be in another package, in helpers doesn´t sound right, maybe data or services.